notatrix/src/utils/constants.ts

```diff
@@ -0,0 +1,40 @@
+"use strict";
```

Comment:
Hm, I guess GitHub thinks this is unrelated to `constants.js`, so it's not showing a side-by-side diff.  The changes here (and in the next few commits) were basically to convert [CommonJS (Node) modules](https://nodejs.org/api/modules.html#moduleexports) to [ES6 modules](https://hacks.mozilla.org/2015/08/es6-in-depth-modules/).  That means

```js
// common.js
exports = {
  a: ...,
  b: ...,
  c: ...,
};
```
becomes
```js
// es6.js
export const a = ...;
export const b = ...;
export const c = ...;
```